src/pages/editor/curators/OrganizationCuratorsAccordion.tsx

```diff
@@ -36,18 +38,32 @@ export const OrganizationCuratorsAccordion = ({
   const [openAddCuratorDialog, setOpenAddCuratorDialog] = useState(false);
   const [expandedState, setExpandedState] = useState(level === 0);
 
-  const isSearchedUnit = organization.id === searchId;
+  const isSearchedUnit = organization.id === unitSearch;
 
-  if (!!searchId && !isSearchedUnit && !includeAllSubunits) {
+  if (!!unitSearch && !isSearchedUnit && !includeAllSubunits) {
     const allSubunits = getAllChildOrganizations(organization.hasPart);
-    if (!allSubunits.some((subunit) => subunit.id === searchId)) {
-      return null; // Hide this element if the searched ID is not a part of this unit
+    if (!allSubunits.some((subunit) => subunit.id === unitSearch)) {
+      return null; // Hide this element if the searched organization ID is not a part of this unit
+    }
+  }
+  if (curatorSearch) {
+    const searchedCurator = curators.filter((curator) => curator.username === curatorSearch);
+    if (searchedCurator.length === 1) {
+      const allSubunits = getAllChildOrganizations([organization]).map((unit) => unit.id);
+      const overlappingOrg = allSubunits.some((unit) => searchedCurator[0].viewingScope.includedUnits.includes(unit));
+      if (!overlappingOrg) {
+        return null; // Hide this element if the searched curator is not a part of this unit
+      }
     }
   }
-  const expanded = expandedState || (!!searchId && !includeAllSubunits);
-  const subunitsCount = organization.hasPart?.length ?? 0;
 
   const curatorsOnThisUnit = curators.filter((curator) => curator.viewingScope.includedUnits.includes(organization.id));
+  const searchedCurators = curatorSearch
+    ? curatorsOnThisUnit.filter((curator) => curator.username === curatorSearch)
+    : curatorsOnThisUnit;
+
+  const expanded = expandedState || (!!unitSearch && !includeAllSubunits) || !!curatorSearch;
```

Comment:
Med denne så vil kuratorsøk alltid være expanded. Kan man på noen måte fjerne `cursor='pointer'` på `<Accordion>` i dette tilfelle? Fant ikke noen lur løsning selv, men kanskje du vet noe? Litt dumt at det ser ut som man kan kollapse menyen når man ikke får lov.

Reply:
Ja, det er en svakhet vi har flere plasser også, feks om du er logget inn i sikt og går til `/institution/overview/organization` så kan du trykke på alle enheter, men det betyr ikke noe 😬 Vet jeg undersøkte det da, uten å komme noen vei.
Tror løsningnen er å rendre `<Paper>` i stedet for `<Accordion>` når den ikke er klikkbar, men det blir fort en del jobb da :/